blocks/card/card.css

```diff
@@ -0,0 +1,36 @@
+.card {
+    overflow: hidden;
+    height: auto;
+}
+
+.card a {
+    text-decoration: none;
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
+}
+
+.card img {
+    transition: all .65s ease;
+}
+
+.card:hover img {
+    transform: scale(1.05);
+    transform-style: preserve-3d;
+}
+
+.card h3 {
+    font-size: var(--body-font-size-s);
+    color: var(--red);
```

Comment:
You can remove the color, and instead add this h3 to styles.css line 515 (where "body.red" is).

Reply:
Done